plugins/workload-management/build.gradle

```diff
@@ -8,19 +8,24 @@
  * Modifications Copyright OpenSearch Contributors. See
  * GitHub history for details.
  */
-
 apply plugin: 'opensearch.opensearchplugin'
 apply plugin: 'opensearch.yaml-rest-test'
 apply plugin: 'opensearch.java-rest-test'
 apply plugin: 'opensearch.internal-cluster-test'
-
 opensearchplugin {
   description = 'OpenSearch Workload Management Plugin.'
   classname = 'org.opensearch.plugin.wlm.WorkloadManagementPlugin'
-  extendedPlugins = [] // Remove autotagging-commons since it's not a plugin
+  extendedPlugins = ['rule-framework']
```

Comment:
One of the engineers should make this type of change and commit to avoid seeing conflicts in everyone's workspace. Noticed similar changes for PRs by @ruai0511 [Lindsay-00](https://github.com/Lindsay-00)

Reply:
I agree Ankit! but without this build will fail hence the change.